views/js/apipCreator/api/accessElementInfo/spoken.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
+ */
+define(['lodash'], function (_) {
+    'use strict';
+
+    var attributes = {
+        "spokenText" : {
+            "type" : 'textNode'
+        },
+        "spokenText.contentLinkIdentifier" : {
+            "type" : 'attribute'
+        },
+        "textToSpeechPronunciation" : {
+            "type" : 'textNode'
+        },
+        "textToSpeechPronunciation.contentLinkIdentifier" : {
+            "type" : 'attribute'
+        },
+        "audioFileInfo.contentLinkIdentifier" : {
+            "type" : 'attribute',
+            "creator" : function (accessElementInfo) {
+                return createAttribute(accessElementInfo, 'audioFileInfo');
+            }
```

Comment:
you may want to create an single function only and pass it as reference for a better readability like that https://gist.github.com/ssipasseuth/e985fc1dc04f1c8191c2